File: assignment 10 - C# practice problems/js/9.js
//Write a JavaScript program to capitalize the first letter of each word of a given string
function capitalizeEachWordDriverFunction() {
    let stringInput = prompt("Enter a string");

    if (stringInput != null) {
        stringInput = stringInput.split(" ");
        let result = "";
        let word = "";

        stringInput.forEach(element => {
            word = element.replace(element.charAt(0), element.charAt(0).toUpperCase());
            result += ` ${word}`;
        });

        alert(result);
    }
}

capitalizeEachWordDriverFunction()

File: assignment 10 - C# practice problems/9.cs
//Write a C# program to capitalize the first letter of each word of a given string.

using System;
namespace MyNameSpace
{
    class Program9
    {
        public static void CapitalizeEachWordDriverFunction()
        {

            Console.WriteLine("Enter a string: ");
            var inputString = Console.ReadLine();
            var split = inputString.Split(" ");
            var result = "";
            foreach (var itemString in split)
            {
                var newString = itemString.ToCharArray();
                newString[0] = char.ToUpper(newString[0]);
                result += (String.Join("", newString) + " ");
            }

            Console.WriteLine(result);
        }
    }
}
